backend/OR-Tools/OR-Tools/Solvers/OrToolsConverter.cs

```diff
@@ -0,0 +1,245 @@
+﻿using Domains.Models.Input;
+using Domains.Models.Output;
+
+using Google.OrTools.ConstraintSolver;
+
+using OR_Tools.Models;
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+
+namespace OR_Tools.Solvers
+{
+    internal class OrToolsConverter
+    {
+        private static readonly int SAME_LOCATION = 0;
+        private static readonly int NO_CROSS_BETWEEN_LOCATION = 1000;
+        
+        private int ToMinutes(DateTime dt) => (dt.Hour * 60) + dt.Minute;
+
+        #region ToWorkDate
+        public WorkData ToWorkData(FileInput fileInput)
+        {
+            DateTime minDate = fileInput.Locations.Select(l => l.From).Min();
+
+            int vehiclesAmount = fileInput.Vehicles.Count();
+            int locationsAmount = fileInput.Locations.Count();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'locationsAmount'.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4635982)